files/justfiles/toggles.just

```diff
@@ -123,9 +123,18 @@ toggle-xwayland ACTION="prompt":
     source /usr/lib/ujust/ujust.sh
     OPTION={{ ACTION }}
     if [ "$OPTION" == "prompt" ]; then
-      echo "${bold}Toggling Xwayland (requires logout)${normal}"
-      echo 'For which DE/WM do you want to toggle Xwayland?'
-      OPTION=$(ugum choose "GNOME" "KDE Plasma" "Sway")
+      IMAGE="$(rpm-ostree status | grep '●')"
+      if [ -n "$(echo $IMAGE | grep 'silverblue')" ] ; then
+          OPTION=gnome
+      elif [ -n "$(echo $IMAGE | grep 'kinoite')" ] ; then
+          OPTION=plasma
```

Comment:
this should be "KDE Plasma" I believe, to match below. Do you mind writing a `.bats` test for this as well?

Reply:
the plasma choice test is `elif [ "$OPTION" == "KDE Plasma" ] || [ "${OPTION,,}" == "plasma" ]; then`. `plasma` passes the 2nd test, after the or. On a somewhat related note, the gnome and sway tests do not need the `[ "$OPTION" == "GNOME" ]` test, because that value also passes the `[ "${OPTION,,}" == "gnome" ]` test.

Reply:
what test do you mean? we don't have a test for this toggle yet

also you're right, I missed that there's already a check for OPTION=plasma below

Reply:
Sorry, I meant condition when I said test.